src/Host.cpp

```diff
@@ -1038,6 +1038,17 @@ void Host::updateConsolesFont()
 {
     if (mpConsole) {
         mpConsole->refreshView();
+
+        TEvent event{};
+        event.mArgumentList.append(qsl("sysSettingChanged"));
+        event.mArgumentTypeList.append(ARGUMENT_TYPE_STRING);
+        event.mArgumentList.append(qsl("font"));
```

Comment:
Should this be main window font? Mini consoles can have their own fonts, as does the mapper info window.